plugins/orchestrator/src/components/ui/OverrideBackstageTable.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {
+  Table as BackstageTable,
+  TableProps,
+} from '@backstage/core-components';
+
+import { makeStyles } from '@material-ui/core';
+
+// Workaround by issue created from overriding the tab theme in the backstage-showcase to add a gray background to disabled tabs.
+// This is achieved by overriding the global Mui-disabled class, which results in the actions column header background turning gray.
+// See https://github.com/janus-idp/backstage-showcase/blob/main/packages/app/src/themes/componentOverrides.ts#L59
+
+const useStyles = makeStyles({
```

Comment:
@batzionb just a question.
What happens if we need something disabled in the Table?

Reply:
I don't think we will want that gray background
That's specific for tabs
The default behavior of disabled of Mui is a different font color

Reply:
Great, thanks for the info!